OP2Helper.h

```diff
@@ -89,3 +89,10 @@ void RandomizeList(int numItems, ListItemType list[])
 
 // Centers the local player's screen on their CC, if they have one.
 void CenterViewOnPlayerCC();
+
+// Use when passing a non-existing unit.
+const Unit NullUnit;
+
+// Set toPlayerNum = -1 to send to all players. 
+// Default sound effect is phone ringing (sending text message to other player)
+void AddMessage(const char* message, int soundIndex = SoundID::sndMessage2, int toPlayerNum = -1, Unit sourceUnit = NullUnit);

```

Comment:
Made toPlayerNum's default -1 instead of 0. I think this makes more sense. In colony games, I suspect -1 will be equivalent to 0. In multiplayer games, the default makes more sense to be all players to me instead of player 0.

Any reason not to use SoundID instead of int for type of soundIndex, and PlayerNum instead of int for type of toPlayerNum?

Thanks,
Brett

Reply:
I like changing the default to `-1` / `PlayerAll`. Let's do that.

Probably also a good idea to be more explicit about types. I like the idea of using `SoundID` and `PlayerNum`. They're regular enums, so they should convert between int. That means the use of enums shouldn't get in the way if someone really wants to use ints somewhere. Might want to test that though.